mava/systems/jax/parameter_client.py

```diff
@@ -129,6 +129,23 @@ def get_async(self) -> None:
             self._copy(self._get_future.result())
             self._get_future = None
 
+    def force_get_async(self) -> None:
+        """Force asynchronously updates the parameters with the latest copy from server.
+
+        Returns:
+            None.
+        """
+        if self._get_future is None:
+            # The update period has been reached and no request has been sent yet, so
+            # making an asynchronous request now.
+            self._get_future = self._async_request()
```

Comment:
Do we need an async case?

Reply:
I will investigate it and give it a try by making it an immediate request

Reply:
Done!

Reply:
Thanks @OmaymaMahjoub :fire: